lib/auth_token/auth_token_creator.rb

```diff
@@ -0,0 +1,41 @@
+class AuthTokenCreator
+  DEFAULT_VALIDITY = 2.hours
+
+  attr_reader :user
+  attr_reader :key
+  attr_reader :expires_at
+
+  def self.create_with_defaults(user)
+    new(user, Rails.application.config.secret_key_base, Time.now + DEFAULT_VALIDITY)
```

Comment:
That's not needed. We're exposing it externally only as UTC to avoid issues with API consumers who use different time zones and DST handling. This is not supposed to be a local time.

```
2.3.7 :011 > Time.zone = "Warsaw"
 => "Warsaw"
2.3.7 :012 > time_now = Time.now; time_zone_now = Time.zone.now
 => Sat, 04 Aug 2018 13:21:39 CEST +02:00
2.3.7 :017 > time_now
 => 2018-08-04 11:21:39 UTC
2.3.7 :018 > time_zone_now
 => Sat, 04 Aug 2018 13:21:39 CEST +02:00 #
2.3.7 :013 > time_now.utc
 => 2018-08-04 11:21:39 UTC
2.3.7 :014 > time_zone_now.utc
 => 2018-08-04 11:21:39 UTC
```